packages/apps/docs/scripts/detect-broken-links.js

```diff
@@ -0,0 +1,182 @@
+const fs = require('fs');
+const path = require('path');
+const http = require('http');
+const https = require('https');
+
+const externalLinks = {}
+
+function getBrokenLinks(filePath, links) {
+  const brokenLinks = [];
+  const directory = path.dirname(filePath);
+  links.forEach((link, index) => {
+    // clean the link of hash fragments
+    link = link.split('#')[0];
+    if (link.length === 0) {
+      return
+    }
+    if (link.startsWith('http')) {
+      if (!externalLinks[filePath]) {
+        externalLinks[filePath] = []
+      }
+
+      externalLinks[filePath].push(link)
+      return;
+    }
+
+    if (link.startsWith('/assets')) {
```

Comment:
asset links can be absolute. so `/assets/blog/FILENAME`
so `../public......` now fails, but the file exists